pkg/controller/config-controller.go

```diff
@@ -232,6 +236,14 @@ func (r *CDIConfigReconciler) reconcileStorageClass(config *cdiv1.CDIConfig) err
 	return nil
 }
 
+func (r *CDIConfigReconciler) reconcileImagePullSecrets(config *cdiv1.CDIConfig) error {
+	if config.Spec.ImagePullSecrets != nil {
+		config.Status.ImagePullSecrets = config.Spec.ImagePullSecrets
+	}
```

Comment:
`config.Status.ImagePullSecrets` will not get unset if `config.Spec.ImagePullSecrets` deleted